src/main/java/mate.academy/lib/Component.java

```diff
@@ -1,5 +1,12 @@
 package mate.academy.lib;
 
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
 public @interface Component {
 
 }
```

Comment:
Empty line, I prefer this format, but I don`t know if it correct
```suggestion
public @interface Component {}
```

Reply:
Let's see what mentor would say

Reply:
both are fine as long as you stay consistent with it